src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Decred DEX Launches DCRDEX 0.6, Offering new levels of Privacy and Security for Cross-Chain Swaps"
+date: 2023-04-18
+sortDate: 2023-04-18
+draft: false
+icon: "announcementsDefault.svg"
+---
+
+**[Chicago, IL, April 17, 2023]** Decred (DCR), a sovereignty and privacy maximizing currency, today released the latest version of its decentralized exchange. DCRDEX 0.6 offers direct peer-to-peer cross-chain swaps without any kind of intermediaries, solving key problems around privacy and security. The combination of privacy, resilience, and cross-chain compatibility in the new release is a first.
+
+New Assets
```

Comment:
```suggestion
#### New Assets

```

Reply:
I think header's visual size (presentation) should be reduced by CSS and not by using a smaller *logical* level. Main title should be level 1 (`#`) and sub headings like New Assets should be level 2 (`##`).

Reply:
Correct. I opened an issue for it.